eng/common/cross/build-rootfs.sh

```diff
@@ -112,6 +112,10 @@ while :; do
             __CodeName=buster
             __LLDB_Package="liblldb-6.0-dev"
             __Keyring="--keyring /usr/share/keyrings/raspbian-archive-keyring.gpg"
+
+            if [[ ! -e "/usr/share/keyrings/raspbian-archive-keyring.gpg" ]]; then
+                __Keyring=
+            fi
```

Comment:
we can clean it up a bit more:
```suggestion

            if [[ -e "/usr/share/keyrings/raspbian-archive-keyring.gpg" ]]; then
                __Keyring="--keyring /usr/share/keyrings/raspbian-archive-keyring.gpg"
            fi
```